core/src/test/java/nu/yona/server/device/service/DeviceServiceTest.java

```diff
@@ -748,27 +750,28 @@ public void getDeviceAnonymizedId_byId_tryGetNonExistingId_exception()
 
 	@Test
 	public void postOpenAppEvent_appLastOpenedDateOnEarlierDay_appLastOpenedDateUpdated()
+			throws IllegalArgumentException, IllegalAccessException
```

Comment:
Can we move setting the app last opened date field inside a helper method so that those `throws` do not have to be repeated?